rust/src/delta.rs

```diff
@@ -1081,16 +1081,36 @@ impl DeltaTable {
         })
     }
 
-    /// Create a DeltaTable with version 0 given the provided MetaData and Protocol
+    /// Create a DeltaTable with version 0 given the provided MetaData, Protocol, and CommitInfo
     pub async fn create(
         &mut self,
         metadata: DeltaTableMetaData,
         protocol: action::Protocol,
+        commit_info: &mut Value,
```

Comment:
better to let the method take the ownership here just like the other arguments like metadata and protocol.
```suggestion
        commit_info: Value,
```

Reply:
Would you also think that commit info should be optional since it isnt necessary? and in that case just create commit info that delta-rs supplies?

Reply:
That's a good idea!